sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersIntegrationTestBase.java

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-package com.azure.communication.administration;
+package com.azure.communication.phonenumbers;
 
 import java.time.OffsetDateTime;
 import java.util.ArrayList;
```

Comment:
We can probably delete the recordings below?

Reply:
Yea you're right, I'll delete the recordings